region/norfair/east/Frog Speedway.json

```diff
@@ -101,6 +101,41 @@
         "h_canCrystalFlash"
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "10 Power Bomb Crystal Flash",
+      "requires": [
+        {"or": [
+          {"resetRoom": {
+            "nodes": [1],
+            "mustStayPut": false
+          }},
+          {"and": [
+            "SpeedBooster",
+            {"resetRoom": {
+              "nodes": [2],
+              "mustStayPut": false
+            }}
+          ]}
+        ]},
+        {"or": [
+          {"enemyDamage": {
+            "enemy": "Beetom",
+            "type": "contact",
+            "hits": 2
+          }},
+          "canInsaneJump"
```

Comment:
Killing them with PBs is rather safe.
Lay PB when the first Beetom comes on screen. Run through the drops before they appear. PB in the corner, pick up drops, run back.

Reply:
Nice. Added it to the description and adjusted the requirements. I guess there's not a comparable strat if you enter from the right and cannot reset the room on the left?

Thinking about the possibility of one-hit-kill mode (or high damage scaling) has been changing how I think about enemy damage requirements, since it means small enemy damage can't be ignored even if there is an available way to farm.

Reply:
You can kill the right 3 beetoms with speed but not break all the speed blocks.  Waste a PB first so you get 3 drops to work with.  More 🤡 than the left door strat.

Reply:
Oh yeah, that works nicely. I think if someone is playing in a "one-hit kill" mode with "can10PowerBombCrystalFlash" then this sort of stuff should be fair game, so probably no need for extra requirements for the safe strats.